tensor([[-0.0592, -0.0085,  0.1322, -0.0578, -0.0215,  0.1351,  0.1015,  0.0389],
        [-0.0596, -0.0084,  0.1323, -0.0575, -0.0214,  0.1352,  0.1013,  0.0392],
        [-0.0594, -0.0085,  0.1322, -0.0577, -0.0214,  0.1351,  0.1014,  0.0390],
        [-0.0595, -0.0084,  0.1323, -0.0576, -0.0214,  0.1351,  0.1013,  0.0392]],
       device='cuda:0', grad_fn=<AddmmBackward>)
Traceback (most recent call last):
  File "train.py", line 38, in <module>
    mu_vector, sigma_vector = policy(state_tensor)
  File "/home/ssbl/anaconda3/envs/ml/lib/python3.7/site-packages/torch/nn/modules/module.py", line 541, in __call__
    result = self.forward(*input, **kwargs)
  File "/home/ssbl/Desktop/PPO/network.py", line 25, in forward
    mu_vector = x[:, 0:self.action]
  File "/home/ssbl/anaconda3/envs/ml/lib/python3.7/site-packages/torch/nn/modules/module.py", line 585, in __getattr__
    type(self).__name__, name))
AttributeError: 'Network' object has no attribute 'action'
